python/README.md

```diff
@@ -3,20 +3,23 @@ MXNet Python Package
 MXNet is a deep learning framework designed for both *efficiency* and *flexibility*.
 It allows you to mix the flavours of deep learning programs together to maximize the efficiency and your productivity.
 
+This directory and nested files contain MXNet Python package and language binding.
 
-Installation
-------------
-To install, check [Build Instruction](http://mxnet.io/get_started/install.html)
+## Installation
```

Comment:
Installation link should be 
https://mxnet.incubator.apache.org/install/index.html

Reply:
Fixed.